packages/superset-ui-core/src/chart/models/ChartProps.ts

```diff
@@ -47,6 +47,8 @@ export interface ChartPropsConfig {
   hooks?: Hooks;
   /** Formerly called "payload" */
   queryData?: QueryData;
+  /** Formerly called "payload" */
+  queriesData?: QueryData[];
```

Comment:
Can we add a comment to `queryData` stating that it only contains the first result and is here to preserve backwards compatibility and will be removed in a future version?